test/e2e/e2e_suite_test.go

```diff
@@ -254,11 +254,50 @@ func initRancherTurtles(clusterProxy framework.ClusterProxy, config *clusterctl.
 	}
 	_, err := chart.Run(map[string]string{
 		"cluster-api-operator.cert-manager.enabled": "true",
```

Comment:
As the proxy flags for capi have a different name, have you considered unified naming patter here?

Reply:
you mean capi-operator vs cluster-api-operator?

Reply:
I've just tried different approaches here, there will still be inconsistency. Helm doesn't like`-` character in go template so `cluster-api.something` is not an option, I changed it to `clusterAPI` and then `cluster-api-operator.enabled` to `clusterAPIOperator.enabled`, and if you want to pass values to capi operator chart you have to use `chartname.something` which is `cluster-api-operator.something`

Reply:
I guess it's better to keep `cluster-api-operator.enabled` to match with other actions related to operator